apis/v1alpha1/gateway_types.go

```diff
@@ -702,12 +698,12 @@ type ListenerStatus struct {
 	// shares the same port value, this message reports the combined
 	// status of all such Listeners.
 	//
-	// +required
+	// +kubebuilder:validation:MaxLength=250
 	Port string `json:"port"`
 
 	// Conditions describe the current condition of this listener.
 	//
-	// +required
+	// +kubebuilder:validation:MaxItems=8
```

Comment:
Should Listener conditions have the [same attributes](https://github.com/kubernetes-sigs/service-apis/pull/336/files#diff-2927d10c91558c2273bc3a8ea6464b85R592-R595) as Gateway conditions?

Reply:
I think it's fine for conditions to be required here since this entry should only exist if there are conditions to list.